server.js

```diff
@@ -191,6 +191,53 @@ var handlerWrapper = module.exports.handlerWrapper = function handlerWrapper(pin
       return;
     }
 
+    //This Week in Servo
+    if (message.indexOf('twis ') > -1) {
+      var command = original_message.split(' ').splice(1);
+      var action = command[1].toLowerCase();
+
+      if (action.indexOf('add') > -1) {
+        if (command.length > 2) { 
+          var message = command.splice(2).join(' ');
+          var twisForUser = twisStorage.getItemSync(from);
+          if (!twisForUser) twisForUser = [];
+
+          twisForUser.push({"from": from, "message": message});
+
+          twisStorage.setItemSync(from, twisForUser);
+          bot.say(to, "Done and done.");
+        } else {
+          bot.say(to, "Sorry. I can't add this. Did you include what you did?");
+        }  
+      } else if (action.indexOf('list') > -1) {       //TODO maybe make it all in one message?
+          bot.say(to, "This Week in Servo!");
+          var twisForUser = twisStorage.getItemSync(from);
```

Comment:
Oh, does this mean that there's a separate TWiS list per user that added items? Like, if both ms2ger and I use `twis add someone did something`, if I use `twis list` I'll only see the entries that I added? If so, we should change that to have a single global store instead.